package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "generator-team-services-extension",
```

Comment:
Probably need to go through https://docs.npmjs.com/cli/deprecate to change the name again :wink: